# Spatial transcriptomic profiling of the aging mouse brain


# Abstract
The mammalian brain can be divided into distinct structural and functional regions to perform a variety of diverse functions, but during normal aging, exactly how each region is affected, and the changes of communication between different regions, remains largely unknown. To gain a better insight into these processes, here we generate a spatial transcriptomic (ST) atlas of young and old mice brains involving cerebrum, brain stem and fiber tracts regions. Based on the unbiased classification of spatial molecular atlas, 27 distinguished brain spatial domains were obtained, some of which were mapped to known anatomical regions, others revealed layer-specific subregions. Through differential expression analysis (DEA) and gene set enrichment analysis (GSEA), we identified aging-related genes and pathways with spatial patterns that vary in a coordinated or opposite manner across regions. Combined with single-cell transcriptomic data, we characterized the spatial distribution of multiple cell types, and found that the proportion of cell types in some regions differs during aging, such as the decrease in the proportion of immN in the DG region. Through ligand-receptor interaction analysis, we identified all possible information interaction changes between regions with aging. In summary, we establish a spatial molecular atlas of aging mouse brain (accessible online at https://8.135.19.221/Mouse-Brian/Aging ) to provide a rich resource of spatially differentially expressed genes and information interaction, which may help to understand aging and provide novel insights into the molecular mechanism of brain aging.

# Data
The data has been uploaded to GSE193107

# Contact
Cheng Wu wucheng820@163.com
